### ISSUE TITLE
Validation of Policies with PolicyTemplates provided by PolicyHub

### ISSUE THREAD

**Issue by mkanal** [eclipse-tractusx/item-relationship-service#452]
**As** ... ,
**I want** ... ,
**so that** ... .

## Hints / Details

- ...

## Acceptance Criteria

- ...

## Out of Scope

- ...

**Comment by mkanal** [comment]
@jzbmw This is the outcome pbi for the data soverignty guardrails for 24.5 please have a look.

**Comment by mkanal** [comment]
BLOCKED as After releasing the blocked by https://github.com/eclipse-tractusx/policy-hub/issues/64 the concept has to be reworked according to new api endpoint
